src/wallet/wallet.cpp

```diff
@@ -47,6 +47,8 @@ OutputType g_change_type = OUTPUT_TYPE_NONE;
 const char * DEFAULT_WALLET_DAT = "wallet.dat";
 const uint32_t BIP32_HARDENED_KEY_LIMIT = 0x80000000;
 
+#define COINCONTROL_AVOID_PARTIAL_SPENDS(cc) ((cc) && (cc)->m_avoid_partial_spends ? (cc)->m_avoid_partial_spends.get() : gArgs.GetBoolArg("-avoidpartialspends", DEFAULT_AVOIDPARTIALSPENDS))
```

Comment:
Seems this would be better as a method on CCoinControl...

Reply:
Coin control needs `gArgs` in that case, and currently there is no `.cpp` file so it'd have to be included in the header.

I'm actually starting to think maybe coin control could use a `.cpp` file...